9.2/run-postgresql.sh

```diff
@@ -16,27 +14,13 @@ else
   export PGDATA=$HOME/data/userdata
 fi
 
-if [ "$1" == "postgres-master" ]; then
-  shift
-  exec /usr/local/bin/run-postgresql-master.sh "$@"
-fi
-
-if [ "$1" == "postgres-slave" ]; then
-  shift
-  exec /usr/local/bin/run-postgresql-slave.sh "$@"
-fi
-
-generate_postgresql_config
-generate_passwd_file
-
-if [ "$1" = "postgres" ]; then
-  if [ ! -f "$PGDATA/postgresql.conf" ]; then
-    initialize_database
-  fi
-  pg_ctl -w start
-  set_passwords
-  pg_ctl stop
-fi
+case "${1:-}" in
+  'postgres-slave' )
```

Comment:
This is backward incompatible change.
Before `docker run ... postgresql-92-centos7 postgres-slave` (or postgres-master) would work, if we change it, it won't.
Existing deployed templates refer to the old name, if they get a new image is breaks the deployment.

If you don't care about anybody using this image in the wild... then it is indeed "simple": change it here and in the template, and perhaps in documentation if there is any.